pylint/config/option_manager_mixin.py

```diff
@@ -137,7 +137,10 @@ def add_option_group(self, group_name, _, options, provider):
 
     def add_optik_option(self, provider, optikcontainer, opt, optdict):
         args, optdict = self.optik_option(provider, opt, optdict)
-        option = optikcontainer.add_option(*args, **optdict)
+        try:
+            option = optikcontainer.add_option(*args, **optdict)
+        except optparse.OptionConflictError:
```

Comment:
@Pierre-Sassoulas Are we sure this doesn't create problems?

Without it for some reason we get an error on `accept-no-param-docs` being a duplicate error. I couldn't figure out why it did this. This solves the issue and passes the tests, but I wonder if this creates other issues..

Reply:
Hmm, it's strange, the error that is caught itself did not seem out of place for me, but if it's ``accept-no-param-docs being a duplicate error`` then it's an internal pylint error not an optparse error ? We should probably investigate if that's the case

Reply:
I found the issue.
It is the extension `check_docs`. Because of it we are double loading `docparams`.

It has been deprecated for 3+ years. Perhaps it is time to remove it?